### DISCUSSION TITLE
i do have some errors

### DISCUSSION THREAD

**Discussion by s11dkl** [m1dugh/RosettaStonks#108] [General]
like this URL.parse is not a function
pls help @m1dugh @AlexandreGirold @kishyassin

**Comment by m1dugh** [comment]
Can you follow the documentation plz ? It is a function if you use the proper browser. That is chrome or firefox.

**Reply by s11dkl** [reply]
it works now but there is no time added
@m1dugh

**Reply by s11dkl** [reply]
@m1dugh 
can u add something to minus time
 plssssssssss

**Reply by m1dugh** [reply]
So I suppose it has worked to add minutes ?

**Reply by s11dkl** [reply]
lmao 
yess its working but i added too much hours, can u pls help mee # @m1dugh

**Reply by m1dugh** [reply]
As already explained multiple times in other discussions, it is not possible to remove time.
